util/chplenv/chpl_qthreads_configs.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import optparse, sys, os, re
+from sys import stdout
+
+import chpl_arch, chpl_compiler, chpl_locale_model, chpl_platform
+import utils
+from utils import memoize
+
+@memoize
+def get_uniq_cfg_path():
+    target_platform = chpl_platform.get('target')
+    target_compiler = chpl_compiler.get('target')
+    target_arch = chpl_arch.get('target', map_to_compiler=True, get_lcd=False)
+    locale_model = chpl_locale_model.get()
+    uniq_cfg_path = '{0}-{1}-{2}-{3}'.format(target_platform, target_compiler,
+                                             target_arch, locale_model)
+    return uniq_cfg_path
+
+
+def handle_la(la_path):
```

Comment:
Or maybe a new third-party-utils.py? I don't expect it to be of general use, and utils.py seems to contain things useful to all the scripts. What do you think?